README.md

```diff
@@ -23,6 +23,8 @@ You can view them at:
 [GOVERNANCE.md](./GOVERNANCE.md)  
 [CONTRIBUTING.md](./CONTRIBUTING.md)
 
+The Evangelism WG is overseen by the [Node.js Community Committee](https://github.com/nodejs/community-committee).
```

Comment:
Nit: Do we have standard language to describe the hierarchical relationship? I would have expected something more like `chartered by` rather than `overseen by`, but I'm not sure that's accurate here.